napari/layers/image/_slice.py

```diff
@@ -147,21 +186,27 @@ def _call_multi_scale(self) -> _ImageSliceResponse:
             ndim=self.dims.ndim,
         )
 
-        thumbnail_indices = self._slice_indices_at_level(self.thumbnail_level)
-
-        image = self.data[level][tuple(indices)]
-        thumbnail = self.data[self.thumbnail_level][tuple(thumbnail_indices)]
+        order = self._get_order()
+        data = np.asarray(self.data[level][tuple(indices)])
+        data = np.transpose(data, order)
+        image = _ImageView.from_view(data)
 
-        if not self.lazy:
-            image = np.asarray(image)
-            thumbnail = np.asarray(thumbnail)
+        thumbnail = image
+        if self.thumbnail_level != level:
```

Comment:
This mirrors some of the computation saving that's done in the other image slice classes and hopefully improves and simplifies it. I.e. when the image and thumbnail are the same data, avoid doing extra work for the thumbnail.